setup/terraform/modules/openai/main.tf

```diff
@@ -29,88 +29,104 @@ resource "azurerm_cognitive_account" "openai" {
   depends_on = [var.keyvault_id]
 }
 
-resource "azurerm_key_vault_secret" "openai_key" {
-  name         = "OPENAI-SERVICE-KEY"
-  value        = azurerm_cognitive_account.openai.primary_access_key
-  key_vault_id = var.keyvault_id
-
-  depends_on = [var.wait_for_propagation]
-}
-
 # A delay is required to avoid a 409 conflict error when adding deployments concurrently
 resource "null_resource" "wait_for_openai_resource" {
   provisioner "local-exec" {
     command = "sleep 60"
   }
-  depends_on = [azurerm_key_vault_secret.openai_key, azurerm_cognitive_account.openai]
+  depends_on = [azurerm_cognitive_account.openai]
 }
 
 resource "azapi_resource" "rai_policy" {
-  type                      = "Microsoft.CognitiveServices/accounts/raiPolicies@2024-04-01-preview"
+  type                      = "Microsoft.CognitiveServices/accounts/raiPolicies@2024-06-01-preview"
   name                      = "Unfiltered"
   parent_id                 = azurerm_cognitive_account.openai.id
   schema_validation_enabled = false
 
   body = jsonencode({
     properties = {
       mode = "Default"
-      basePolicyName : "Microsoft.Default",
+      basePolicyName : "Microsoft.DefaultV2",
       contentFilters : [
         {
-          name : "hate",
-          allowedContentLevel : "Medium",
-          blocking : false,
-          enabled : false,
-          source : "Prompt"
+          "name" : "Violence",
+          "severityThreshold" : "Low",
+          "blocking" : false,
+          "enabled" : true,
```

Comment:
What is the benefit of having these non-blocking, but enabled content filters? Are we doing anything with the outputs of the content filter? If not I would rather just see the filters disabled.